packages/flutter_test/lib/src/window.dart

```diff
@@ -243,6 +243,13 @@ class TestWindow implements Window {
     _window.onDrawFrame = callback;
   }
 
+  @override
+  TimingsCallback get onReportTimings => _onReportTimings;
```

Comment:
The analyzer is flagging this.  Should this return `_window.onReportTimings`?

Reply:
Thanks! Fixed.